src/garage/torch/algos/loss_function_utils.py

```diff
@@ -1,15 +1,61 @@
-"""loss function utilities."""
+"""Differentiable utils typically used when computing loss functions."""
 import torch
 import torch.nn.functional as F
 
 
-def compute_advantages(discount, gae_lambda, max_len, baselines, rewards):
-    """Calculate advantages."""
-    filter = torch.full((1, 1, 1, max_len - 1), discount * gae_lambda)
+def compute_advantages(discount, gae_lambda, max_path_length, baselines,
+                       rewards):
+    """
+    Calculate advantages.
+
+    Calculate advantages using a baseline (value function) according to
+    Generalized Advantage Estimation (GAE)
+    The discounted cumulative sum can be computed using conv2d with filter.
+    filter:
+        [1, (discount * gae_lambda), (discount * gae_lambda) ^ 2, ...]
+        where the length is same with max_path_length.
+
+    Advantages are a discounted cumulative sum.
+
+    if paths is consisted with
```

Comment:
why mention paths when they are not an argument to the function?

Reply:
delete the comment